src/main/java/com/woowacamp/soolsool/core/cart/dto/request/CartItemModifyRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.woowacamp.soolsool.core.cart.dto.request;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+import lombok.Getter;
+
+@Getter
+public class CartItemModifyRequest {
+
+    @NotBlank(message = "수량은 필수 입력 값입니다.")
+    @Positive(message = "수량은 양수만 요청할 수 있습니다.")
+    public final Integer liquorQuantity;
+
+    @JsonCreator
+    public CartItemModifyRequest(@JsonProperty final Integer liquorQuantity) {
+        this.liquorQuantity = liquorQuantity;
+    }
```

Comment:
```suggestion
    @JsonCreator
    public CartItemModifyRequest(final Integer liquorQuantity) {
        this.liquorQuantity = liquorQuantity;
    }
```
Java 8 이후부터 compiler 옵션을 통해 parameter 이름에 접근할 수 있게 변경되었습니다. 그래서 `@JsonProperty`는 사용안해도 돼요! `@JsonCreator`를 다는 기준은 역직렬화 입니다. 역직렬화가 필요한 경우 사용하고 그렇지 않은 경우는 달지 않아도 돼요.
 
예를 들어 request dto는 `@JsonCreator`를 달아야하고, response dto는 달지 않아도 됩니다. `@JsonProperty`는 다 안 달아도 되구요.
 
 관련된 내용이 더 궁금하면 [이 글](https://github.com/whatasame/TIL/blob/main/stub/jvm-language/library/jackson/%EC%97%AD%EC%A7%81%EB%A0%AC%ED%99%94%EC%97%90%20%EC%8B%A4%ED%8C%A8%ED%95%98%EB%8A%94%20%EA%B2%BD%EC%9A%B0.md)을 참고해주세요